client/run-telepresence.sh

```diff
@@ -18,6 +18,8 @@
 
 set -e
 
+# script directory
+SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
```

Comment:
```suggestion
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

See: https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script